api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -236,6 +236,9 @@ message AccessLogCommon {
   // For HTTP: Total number of bytes received from the upstream by the http stream.
   // For TCP: Total number of bytes sent to the upstream by the tcp proxy.
   uint64 upstream_wire_bytes_received = 32;
+
+  // The type of the access log, which indicates when the log was recorded.
+  string access_log_type = 33;
```

Comment:
`enum` would be better for this type field. And this would deprecate the `intermediate_log_entry` field, right?

Reply:
Thanks, I'll switch to enum. Does it have to deprecate ``intermediate_log_entry`` though? seems like they can co-exist